src/main/java/at/hannibal2/skyhanni/events/BlockClickEvent.kt

```diff
@@ -4,7 +4,9 @@ import at.hannibal2.skyhanni.data.ClickType
 import at.hannibal2.skyhanni.utils.BlockUtils.getBlockStateAt
 import at.hannibal2.skyhanni.utils.LorenzVec
 import net.minecraft.item.ItemStack
+import net.minecraftforge.fml.common.eventhandler.Cancelable
 
+@Cancelable
 class BlockClickEvent(val clickType: ClickType, val position: LorenzVec, val itemInHand: ItemStack?) : LorenzEvent() {
```

Comment:
This does not extend world click event. is this intended?

Reply:
that is not intended. Happend during merge conflict sloven. (Also the bug with the block clicking)